### DISCUSSION TITLE
Arrow keys issue on q-select popup when max-height is added to .q-virtual-scroll__content

### DISCUSSION THREAD

**Discussion by rebenely** [quasarframework/quasar#16073] [General - Components / Directives / etc] [answered]
I'm trying to add a max-height to a q-select dropdown with an after-options slot element.
The after-options div should always be displayed regardless of the scroll position.

I've tried two approaches:

- Make the div sticky so that it's always shown, but the sticky div shouldn't be included in the scroll. So I tried another approach;
- Add max-height to `.q-virtual-scroll__content`. This is what I want to see visually, but I have issues when scrolling using keys.

https://codepen.io/rebenely/pen/VwVKQrL

In both cases, they have issues when using arrow keys to move the selection.

In the first approach, the keys can move the selection properly but the last row is blocked by the sticky div.
In the second approach, the keys move the selection but it doesn't scroll.

Any help will be greatly appreciated. Thanks.

**Comment by pdanpdan** [comment, marked as answer]
The first approach is the right one, you needed to add `virtual-scroll-sticky-size-end="61"`
https://codepen.io/pdanpdan/pen/LYXQdyN?editors=1100

**Reply by rebenely** [reply]
That works, thanks! 

Though is there a way to make the second approach work?

**Reply by pdanpdan** [reply]
NOOO :)
That way you break all presumptions on which virtual-scroll is built (the set of options displayed are scrolling instead of the container with content and padding)

The first approach is implemented in virtual-scroll

**Reply by rebenely** [reply]
That figures. Thanks again!
